src/components/who-is-online/index.ts

```diff
@@ -40,11 +41,30 @@ export class WhoIsOnline extends BaseComponent {
    */
   protected destroy(): void {
     this.unsubscribeToRealtimeEvents();
+    this.removeListeners();
     this.element.remove();
     this.element = null;
     this.participants = null;
   }
 
+  /**
+   * @function addListeners
+   * @description adds event listeners to the who is online element.
+   * @returns {void}
+   */
+  private addListeners(): void {
+    this.element.addEventListener('go-to-mouse-pointer', this.goToMousePointer);
+  }
+
+  /**
+   * @function addListeners
+   * @description adds event listeners from the who is online element.
+   * @returns {void}
+   */
+  private removeListeners(): void {
+    this.element.removeEventListener('go-to-mouse-pointer', this.goToMousePointer);
+  }
```

Comment:
You can use realtime events here, it's a good thing because the go-to in WIO would also work with 3d plugins